### ISSUE TITLE
Get familiar with fleet management system

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#8]
## User story
1. As a SD
2. I want / need to get used to the fleet management system
3. So that I can get start in my topic

## Acceptance criteria
* Good knowledge

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
